saleor/payment/gateways/authorize_net/tests/test_plugin.py

```diff
@@ -101,7 +101,7 @@ def test_payment_gateway_refund_payment(
 def test_payment_gateway_refund_payment_no_payment(
     authorize_net_plugin, dummy_payment_data
 ):
-    dummy_payment_data.payment_id = 100
+    dummy_payment_data.payment_id = -1
```

Comment:
Maybe?
```suggestion
    dummy_payment_data.payment_id = None
```

Reply:
IMO, we should rather have tests for both situations:
- when `payment_id` is empty,
- when `payemnt_id` has value, but payment with this `id` does not exist